company.el

```diff
@@ -537,9 +537,10 @@ even if the backend uses the asynchronous calling convention."
 (defcustom company-transformers nil
   "Functions to change the list of candidates received from backends.
 
-Each function gets called with the return value of the previous one.
-The first one gets passed the list of candidates, already sorted and
-without duplicates."
+Each function is called with the return value of the previous one.  The
```

Comment:
Could you reformat this by hand so that "The" goes to the next line?

Reply:
Sure. Done.